index.html

```diff
@@ -311,31 +311,33 @@ <h2 id="heading">About me</h2>
       <ul id="second-media" class="flexs">
         <li>
           <a class="icons" href="https://github.com/leehaney254"
-            ><img src="./assets/gitfoot.png" alt="Github link"
+            ><img src="./assets/gitfoot.png" alt="Link to my githb page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://www.linkedin.com/in/leehaney-george-0a4a51178/"
-            ><img src="./assets/linkedinfoot.png" alt="Linkedin icon"
+            ><img
+              src="./assets/linkedinfoot.png"
+              alt="Link to my linkedin page"
```

Comment:
Modified the image alt text to be more descriptive and null for decoration. :100: